### PR TITLE
Decode character entities given in descriptions.

### PR DESCRIPTION
**Pull request by toddjordan**
This is a patch for https://github.com/emberjs/ember.js/issues/18063 which lets us provide yuidoc special characters as html character code entities and decodes them before we apply highlightjs and markdown processing.

Here's a preview of the screen for the above issue
image

### CODE REVIEW

**Comment by jenweber** [comment]
Waiting for @sivakumar-kailasam to chime in before we merge this

**Comment by sivakumar-kailasam** [comment]
This is exactly what I had in mind.🙏 @toddjordan could you make the test pass before merging this though?

**Review by sivakumar-kailasam — APPROVED** [review]

**Comment by toddjordan** [comment]
@sivakumar-kailasam so husky was preventing the build from passing because it was adjusting the indentation of a switch statement in the file, so for now I'm removing husky so we can get this in.

**Comment by toddjordan** [comment]
if someone has a husky config that plays well with our eslint config we can re-enable it

**Comment by sivakumar-kailasam** [comment]
SGTM

**Pull request merged**
